### ISSUE TITLE
Slack integration doesn't work

### ISSUE THREAD

**Issue by bogdanblinov** [grafana/oncall#1303]
Hi, Team! I’ve connected self-hosted grafana to the slack using this [guide](https://grafana.com/docs/oncall/latest/open-source/), configured alertmanager and escalation chains, but I don’t receive any messages from grafana oncall in slack. There are no any errors in UI and I’m using helm chart (1.1.22) and frontend ui plugin (1.1.22).

When I open slack connection page from the oncall plugin page and push the button "allow" for ONCALL bot, the page is freezed and nothing happens. But when I refresh oncall plugin page it looks like slack is succesfully connected. 
Logs from engine pod:
```
2023-02-07 10:28:24 source=engine:uwsgi status=200 method=GET path=/api/internal/v1/current_team latency=0.035207 google_trace_id=035f580242a25d0acec4991919f12ec4/13605114600599088258 protocol=HTTP/1.1 resp_size=813 req_body_size=0
2023-02-07 10:28:26 source=engine:app google_trace_id=3dea401db56f21722523f16e0943768c/3264451696074585737 logger=root inbound latency=0.329594 status=200 method=GET path=/api/internal/v1/login/slack-install-free/ content-length=0 slow=0 user_id=2 org_id=1 
2023-02-07 10:28:26 source=engine:uwsgi status=200 method=GET path=/api/internal/v1/login/slack-install-free/ latency=0.331128 google_trace_id=3dea401db56457822523f16e0943768c/3264439456074585737 protocol=HTTP/1.1 resp_size=1027 req_body_size=0
2023-02-07 10:28:32 source=engine:app google_trace_id=fbf5355603450c572a7578908a194523/26634444345292088 logger=apps.slack.models.slack_team_identity updated oauth_fields for sti 1
2023-02-07 10:28:32 source=engine:app google_trace_id=fbf535eb340c572a3678908a194523/34435147345292088 logger=apps.slack.models.slack_user_identity Update user profile info
2023-02-07 10:28:32 source=engine:app google_trace_id=fbf535eb3456456672a7578908a194345/26600154562292088 logger=root inbound latency=1.414728 status=302 method=GET path=/api/internal/v1/complete/slack-install-free/ content-length=0 slow=0 user_id=2 org_id=1 
2023-02-07 10:28:32 source=engine:uwsgi status=302 method=GET path=/api/internal/v1/complete/slack-install-free/?slack_login_token=8489f6358fghb5c8c544407fhg605e09f7a80afed0a4rdf789f66802562c66c&code=1884578991.4798743287442.70ee1566f6dop12683a8ed3e4b9adc983c1794a33cce8e4cbn46bdb4c090eb10&state= latency=1.416340 google_trace_id=fbf535eb6345c5754478908a194523/2660456632292088 protocol=HTTP/1.1 resp_size=464 req_body_size=0
2023-02-07 10:29:01 source=engine:app google_trace_id=7706bf98123fc766c196a350bb9b39ec/16612316892331141003 logger=apps.slack.views Request id: None
2023-02-07 10:29:01 source=engine:app google_trace_id=7706bf98259fc123c196a350bb9b39ec/16612316892331141003 logger=apps.slack.views team_id: T02HT9YV5 channel_id: None user_id: None command: None event: user_change
2023-02-07 10:29:01 source=engine:app google_trace_id=7706bf9345459fc766c196a350bb9b39ec/16612316892331141003 logger=apps.slack.views Organization: None
2023-02-07 10:29:01 source=engine:app google_trace_id=7706bf98259fc766c196a350bb9b39ec/16612316892331141003 logger=apps.slack.views SlackUserIdentity detected: None
2023-02-07 10:29:01 source=engine:app google_trace_id=7706bf98259fc766c123a350bb9b39ec/16614916892456141003 logger=apps.slack.views Event user_change. Dropping request because it does not have SlackUserIdentity.

```

**Comment by Matvey-Kuk** [comment]
What happens when you connect slack channels & try sending alerts to Slack?

**Comment by bogdanblinov** [comment]
I've tried to send demo alerts, but nothing happened. The channel in slack is empty, there are not any indications that I've added oncall app to this slack channel. But from UI side, slack is connected, I don't see any warnings related to connection with Slack. 

![image](https://user-images.githubusercontent.com/115159101/217597035-7309ab71-cab3-41e9-a925-be1d88319e54.png)

![image](https://user-images.githubusercontent.com/115159101/217596991-404f08af-bcf0-40da-997e-b4f561900867.png)

![image](https://user-images.githubusercontent.com/115159101/217597092-2de506b6-b0f3-4971-8e47-d685a90e9506.png)
